Trabalho 2/entities/warehouse/main.py

```
@@ -16,60 +22,67 @@ def on_connect(client, userdata, flags, reason_code, properties):
+                client.publish("estoque/status", f"{part_name}:OUT_OF_STOCK")
             
             # Mostra o status do estoque
-            print(f"[WAREHOUSE] Novo status do estoque: {buffer}") 
+            print(f"[WAREHOUSE] Novo status do estoque: {buffer}")
+            # Publica a atualização para o dashboard
+            client.publish(f"dashboard/inventory/{part_name}", f"{buffer.current_quantity}:{buffer.status}")
 
             # Verifica se precisa reabastecer
-            if buffer.status == "VERMELHO" and not restock_ordered:
+            if buffer.status == "VERMELHO" and not restock_ordered[part_name]:
                 print(f"[WAREHOUSE] NÍVEL CRÍTICO ATINGIDO. Solicitando reabastecimento ao fornecedor.")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)